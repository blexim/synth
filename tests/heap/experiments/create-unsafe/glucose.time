WARNING! DIMACS header mismatch: wrong number of variables.
Command exited with non-zero status 10
0.16user 0.00system 0:00.17elapsed 98%CPU (0avgtext+0avgdata 16696maxresident)k
0inputs+16outputs (0major+5383minor)pagefaults 0swaps
