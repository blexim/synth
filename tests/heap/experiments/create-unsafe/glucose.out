c
c This is glucose 4.0 --  based on MiniSAT (Many thanks to MiniSAT team)
c
c ========================================[ Problem Statistics ]===========================================
c |                                                                                                       |
c |  Number of variables:         23174                                                                   |
c |  Number of clauses:           88994                                                                   |
c |  Parse time:                   0.03 s                                                                 |
c |                                                                                                       |
c | Preprocesing is fully done
c |  Eliminated clauses:           0.79 Mb                                                                |
c |  Simplification time:          0.11 s                                                                 |
c |                                                                                                       |
c ========================================[ MAGIC CONSTANTS ]==============================================
c | Constants are supposed to work well together :-)                                                      |
c | however, if you find better choices, please let us known...                                           |
c |-------------------------------------------------------------------------------------------------------|
c |                                |                                |                                     |
c | - Restarts:                    | - Reduce Clause DB:            | - Minimize Asserting:               |
c |   * LBD Queue    :     50      |   * First     :   2000         |    * size <  30                     |
c |   * Trail  Queue :   5000      |   * Inc       :    300         |    * lbd  <   6                     |
c |   * K            :   0.80      |   * Special   :   1000         |                                     |
c |   * R            :   1.40      |   * Protected :  (lbd)< 30     |                                     |
c |                                |                                |                                     |
c ==================================[ Search Statistics (every  10000 conflicts) ]=========================
c |                                                                                                       |
c |          RESTARTS           |          ORIGINAL         |              LEARNT              | Progress |
c |       NB   Blocked  Avg Cfc |    Vars  Clauses Literals |   Red   Learnts    LBD2  Removed |          |
c =========================================================================================================
c last restart ## conflicts  :  434 91 
c =========================================================================================================
c restarts              : 5 (258 conflicts in avg)
c blocked restarts      : 0 (multiple: 0) 
c last block at restart : 0
c nb ReduceDB           : 0
c nb removed Clauses    : 0
c nb learnts DL2        : 97
c nb learnts size 2     : 41
c nb learnts size 1     : 4
c conflicts             : 1292           (7744 /sec)
c decisions             : 4731           (0.00 % random) (28357 /sec)
c propagations          : 162346         (973064 /sec)
c conflict literals     : 14028          (14.89 % deleted)
c nb reduced Clauses    : 23
c CPU time              : 0.16684 s

s SATISFIABLE
